CMakeLists.txt

```diff
@@ -326,7 +326,7 @@ Option(DOXYGEN_GENERATE_DOCUMENTATION "Build documentation for TUM OpenInfraPlat
 if(DOXYGEN_GENERATE_DOCUMENTATION)
 
 	find_file(DOXYGEN_EXECUTABLE NAMES doxygen.exe HINTS
-		"C:/thirdparty/doxygen"
+		"C:/thirdparty/doxygen/bin"

```

Comment:
Why this change?

Reply:
I reverted it here: 5c04f9e . I will add an Issue and a new Pull request regarding doxygen on Saturday